susemanager-utils/cluster-providers/caasp/caasp/kill_ssh_agent.sls

```diff
@@ -0,0 +1,8 @@
+{%- if not salt['pillar.get']('params:use_ssh_agent') %}
+mgr_caasp_kill_agent:
+  module.run:
+    - name: ssh_agent.kill
+    - order: last
```

Comment:
Is `order: last` really needed? Can we use more require statements instead?

The reason is, that the only state where we use `order: last` is, updating and starting salt-minion.
If possible, please prevent this usage.

Reply:
This can be included in several other cluster `sls` files. The problem is that it doesn't know in which files it's being included. For now I think we can keep it like this and think about a better solution in the next iterations. I don't think this will conflict with updating and restarting salt-minon.

Reply:
ok